git-sync/main.go

```diff
@@ -66,8 +84,24 @@ func main() {
 	if _, err := exec.LookPath("git"); err != nil {
 		log.Fatalf("required git executable not found: %v", err)
 	}
+
+	if *flUsername != "" && *flPassword != "" {
+		log.Println("creating .netrc file (required for https authentication)")
+		if err := createNetrcFile(*flUsername, *flPassword, *flRepo); err != nil {
+			log.Fatalf("error creating .netrc file: %v", err)
+		}
+	}
+
+	if *flOneTime {
```

Comment:
rather than a new block, can you test this flag inside the for loop and exit(0), rather than duplicating it into a whole block like this?